workspaces/lightspeed/plugins/lightspeed/config.d.ts

```diff
@@ -38,11 +38,6 @@ export interface Config {
        */
       url: string;
     }>;
-    /**
-     * query restriction validation (Enabled by default).
-     * @visibility frontend
-     */
-    questionValidation?: boolean;
```

Comment:
we can remove this as part of frontend PR so that all the frontend changes can be released together as major release, wdyt?